pkg/cli/public_key_server_test.go

```diff
@@ -110,6 +110,8 @@ func TestPublicKeyServerCommand(t *testing.T) {
 				t.Fatal(err)
 			}
 
+			req.Header.Set("Accept", "application/json")
+
```

Comment:
maybe not needed?

Reply:
The test doesn't check body content; only status code. Should be ok to remove.